plugins/train/_config.py

```diff
@@ -68,23 +60,249 @@ def set_globals(self):
+                 "statistics, and translation invariance than MAE.\n"
+                 "http://www.cns.nyu.edu/pub/eero/wang03-reprint.pdf\n"
+                 "\nGMSD ---\n"
+                 "Gradient magnitude similarity deviation (GMSD). The image pixel \n"
+                 "to pixel gradients are sensitive to image distortions, where \n"
+                 "different local structures in a distorted image suffer different \n"
+                 "degrees of degradations. Global variation of gradient based local \n"
+                 "quality maps are used for overall image quality prediction. The \n"
+                 "pixel-wise gradient magnitude similarity (GMS) between the reference \n"
+                 "and distorted images combined with a novel pooling strategy – the \n"
+                 "standard deviation of the GMS map – can predict perceptual image \n"
+                 "quality accurately."
+                 "https://arxiv.org/ftp/arxiv/papers/1308/1308.3052.pdf\n"
+                 "\nPixel_Gradient_Difference ---\n"
+                 "Instead of minimizing the difference between the absolute value of \n"
+                 "each pixel in two reference images, compute the pixel to pixel \n"
+                 "spatial difference in each image and then minimize the difference \n"
+                 "between these two gradient maps. Allows for large color shifts,but \n"
+                 "maintains the structure of the image."
+                 )
+        self.add_item(section=section, title="mask_type", datatype=str, default="none",
+                      choices=get_available_masks(),
+                      info="The mask to be used for training:"
+                           "\n\t none: Doesn't use any mask."
+                           "\n\t components: An improved face hull mask using a facehull of 8 "
+                           "facial parts"
+                           "\n\t dfl_full: An improved face hull mask using a facehull of 3 "
+                           "facial parts"
+                           "\n\t extended: Based on components mask. Extends the eyebrow points "
+                           "to further up the forehead. May perform badly on difficult angles."
+                           "\n\t facehull: Face cutout based on landmarks")
+        self.add_item(
+            section=section, title="learning_rate", datatype=float, default=5e-5,
+            min_max=(1e-6, 1e-4), rounding=6, fixed=False,
+            info="Learning rate - how fast your network will learn (how large are \n"
+                 "the modifications to the model weights after one batch of training).\n"
+                 "Values that are too large might result in model crashes and the \n"
+                 "inability of the model to find the best solution.\n"
+                 "Values that are too small might be unable to escape from dead-ends \n"
+                 "and find the best global minimum.")
+
+        # << ORIGINAL MODEL OPTIONS >> #
```

Comment:
Bad news. All of these model specific options need to go. They are now handled in the relevant _default.py with the model

Reply:
Gah - I'll look at how to integrate with the new method

Reply:
Fortunately I have documented them extensively!